_observing-your-data/event-analytics.md

```diff
@@ -24,33 +24,61 @@ For example, the following PPL query returns a count of how many host addresses
 
 After Dashboards generates a visualization, you must save it if you want to return to it at a later time or if you want to add it to an [operational panel]({{site.url}}{{site.baseurl}}/observing-your-data/operational-panels).
 
 To save a visualization, expand the save dropdown menu next to **Refresh**, enter a name for your visualization, then choose **Save**. You can reopen any saved visualizations on the event analytics page.
 
-## View logs
+## Adding event analytics visualizations to dashboards
+
+Presenting your visualizations on a dashboard, instead of the event analytics page, makes it easier for users with varying levels of technical knowledge to understand and interpret the data at a glance. To create a new visualization and add it to a dashboard, follow these steps:
+
+1. On the main menu, select **Observability** > **Event analytics**.
+2. On the Event analytics window, enter the index source in the PPL query field, for example, `source = opensearch_dashboards_sample_data_logs | stats max(bytes), avg(bytes) by host`.
+3. Set the time filter, for example **Week to date**, then select **Refresh**.
+4. Choose the visualization type, for example **Time series**, from the right sidebar dropdown menu.  
+5. Select **Save** and enter a name for the visualization. To add the visualization to an existing dashboard, choose from the **Custom operational dashboards/application** dropdown menu. 
+6. Select **Save**.
+
+You've created a new visualization that can be added to an existing or new dashboard. To work with the dashboard view, follow these steps:  
+
+1. On the main menu, select **Dashboard**.
+2. On the **Dashboards** window, select **Create dashboard**.
+3. On the Dashboards window, select **Add existing** and then choose the visualization.
+4. To add more visualizations to the dashboard, select **Create new** and then select **PPL** on the New Visualization window. You'll return to the event analytics page and follow steps 1-6 in the preceding instructions.
+
+This feature is available in OpenSearch Dashboards version 2.7 and later. It works with new visualizations created in version 2.7 or later that use PPL to query data from OpenSearch or federated data sources such as Prometheus.
+{: .note}
+
+### Limitations of PPL visualizations
+
+PPL visualizations currently do not support [Dashboards Query Language (DQL)]({{site.url}}{{site.baseurl}}/dashboards/discover/dql/) or [query domain-specific language (DSL)]({{site.url}}{{site.baseurl}}/query-dsl/) and they do not use index patterns. Note the following limitations: 
+
+- PPL visualizations will only use filters created using the dropdown interface. If you have DQL query or DSL filters in a dashboard, PPL visualizations will not use them.
+- The **Dashboard** filter dropdown will only show fields from the index patterns used by other visualizations in the same dashboard (or from the default index pattern if PPL visualizations are the only visualization types in the dashboard).
```

Comment:
I realized this might not be accurate as there can be non-PPL visualizations but also not using index patterns, like visbuilder in 2.6 which had the same limitation

```suggestion
- The **Dashboard** filter dropdown interface will only show fields from the default index pattern or index patterns used by other visualizations in the same dashboard.
```